src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -62,17 +63,13 @@ default void deleteAll() {
         getRepository().deleteAll();
     }
 
-    default boolean renameFilter(String name, String newName) {
-        if (getRepository().existsById(name)) {
-            getRepository().rename(name, newName);
-            return true;
-        }
-        return false;
+    default boolean renameFilter(UUID id, String newName) {
+        return getRepository().rename(id, newName) != 0;
     }
 
-    default boolean deleteById(String name) {
-        if (getRepository().existsById(name)) {
-            getRepository().deleteById(name);
+    default boolean deleteById(UUID id) {
+        if (getRepository().existsById(id)) {
+            getRepository().deleteById(id);
```

Comment:
Maybe the repository deleteById method could be derived from base class and return an integer containing the number of deleted records (like the update method), to avoid the call to existsById ?